modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/action/executor/SplitCommerceOrderByCatalogObjectActionExecutorImpl.java

```diff
@@ -219,94 +221,113 @@ private void _createSupplierOrders(
 		RoundingMode roundingMode = RoundingMode.valueOf(
 			commerceCurrency.getRoundingMode());
 
-		for (List<CommerceOrderItem> commerceOrderItemList :
-				commerceOrderItemMap.values()) {
+		commerceOrderItemMap.forEach(
+			(commerceCatalog, commerceOrderItems) -> {
+				CommerceOrder supplierCommerceOrder =
+					customerCommerceOrder.cloneWithOriginalValues();
```

Comment:
why cloneWithOriginalValues() instead of clone()?

Reply:
I don't remember why, it's been there since the first version of the code I committed.
Do you want me to change it?